index.html

```diff
@@ -2755,6 +2755,19 @@ <h3>
 associated with a cryptographic hash of the content of the JSON-LD Context as
 part of registration in the DID Specification Registries [[?DID-SPEC-REGISTRIES]].
             </p>
+            <p class="advisement">
+              If included, the cryptographic hash of the content of the JSON-LD Context MUST be computed in a manner equivalent to the following:
```

Comment:
white space, line length
```suggestion
If included, the cryptographic hash of the content of the JSON-LD Context MUST be
computed in a manner equivalent to the following:
```